documentation/supported-tags.md

```diff
@@ -1,8 +1,17 @@
 # Supported Tags
 
-This document describes the tagging practices used with the official .NET Framework Docker images.
+This document describes the tagging patterns and policies that are used for the official .NET Framework container images.
+.NET tags are intended to closely match the tagging patterns used by [Official Images on Docker Hub](https://hub.docker.com/search?q=&type=image&image_filter=official).
+ Please [log an issue](https://github.com/dotnet/dotnet-docker/issues/new/choose) if you encounter problems using .NET images or applying these tagging patterns.
 
-The .NET image tags strive to align with the tagging practices utilized by the [Official Images on Docker Hub](https://hub.docker.com/search?q=&type=image&image_filter=official).
+Complete tag lists:
+
+* [sdk](../README.sdk.md#full-tag-listing): .NET Framework SDK
+* [aspnet](../README.aspnet.md#full-tag-listing): ASP.NET Web Forms and MVC
+* [runtime](../README.runtime.md#full-tag-listing): .NET Framework Runtime
+* [wcf](../README.wcf.md#full-tag-listing): Windows Communication Foundation (WCF)
+* [samples](../README.samples.md#full-tag-listing): .NET Framework, ASP.NET and WCF Samples
+* [Microsoft Artifact Registry](https://mcr.microsoft.com/en-us/catalog?search=dotnet/framework)
```

Comment:
This is not a tags list. If you want to keep this link, then perhaps line 7 could be something like

```diff
-Complete tag lists:
+Complete tag lists are available on the [Microsoft Artifact Registry](https://mcr.microsoft.com/en-us/catalog?search=dotnet/framework):
```

Reply:
I copied this from dotnet-docker.  I don't like it and am removing it.  I will open a PR against dotnet-docker once this work is complete.